src/components/site-header/styles/_Header.scss

```diff
@@ -1,34 +1,21 @@
 @import './Menu.scss';
 
+$header-logo-height-desktop: 1.75rem;
+$header-logo-height-mobile: 1.5rem;
+
 .site-header-mobile,
 .site-header-desktop {
   background: $white;
   position: relative;
   z-index: 1000;
 
-  .logo {
-    margin-right: 1rem;
-  }
-
   .dropdown-item a {
     text-decoration: none;
   }
 
-  a.logo img {
-      max-height: 50px;
-  }
-
-  .logo-right {
+  .vertical-divider {
     border-left: 1px solid $gray-200;
   }
-
-  .partner-header-logo {
-    width: 70px;
```

Comment:
Note: generally a good idea to scale images for the header with `height` versus `width`. Depending on the logo image (e.g., if/when we support future course type partner logos), using a `width` could lead to a overly tall image compared to the primary logo.